app/src/main/java/org/fossasia/openevent/app/module/event/list/EventsListAdapter.java

```diff
@@ -77,6 +80,7 @@ public int getItemCount() {
         public void bind(Event event) {
             this.event = event;
             binding.setEvent(event);
+            binding.setSelectedEventId(selectedEvent == null ? -1 : selectedEvent.id);
```

Comment:
Similarly here, you don't need to compute it on each binding, this can be stored as the only required variable